README.md

```diff
@@ -39,14 +47,16 @@ Step 2 should have provided the appropriate credentials to push the image to qua
 
 #### Enable quay.io image push for [build.yaml](https://github.com/2i2c-org/hub-user-image-template/blob/main/.github/workflows/build.yaml)
 
-The [build.yaml](https://github.com/2i2c-org/hub-user-image-template/blob/main/.github/workflows/build.yaml) workflow builds the container image and pushes it to quay.io **if** credentials and image name are properly set. This happens on every pushed commit on the main branch of the repo.
+The [build.yaml](https://github.com/2i2c-org/hub-user-image-template/blob/main/.github/workflows/build.yaml) workflow builds the container image and pushes it to quay.io **if** credentials and image name are properly set. This happens on every pushed commit on the main branch of the repo (including when a PR is merged).
 
 To enable pushing to the appropriate quay.io repository, edit line 35 of [build.yaml](https://github.com/2i2c-org/hub-user-image-template/blob/main/.github/workflows/build.yaml#L34-L35) and:
 
 * uncomment the `IMAGE_NAME` option
 * replace `<quay-username>/<repository-name>` with the info of the `quay.io` repository created at step 2
 * commit the changes you've made to `build.yaml`
 
+If you want to see the log of the Github Action that results you can see this via the Github Actions tab on your image repository. If you are triggering this action by merging a PR you should look at the Github Actions tab (and this will show "pushing quay.io/..."). The build log linked from the PR page is not the one for the merge, but the build done for the PR, prior to the merge (and will therefore, by default, not show "pushing quay.io/..."). 
```

Comment:
We may want to add something like "... and from the left hand sidselect 'Build container image'   to access logs that push the image, or 'Test container image build' for logs that do not push the image" to this section. Like from the below screenshot.

<img width="724" alt="Screenshot 2022-04-07 at 10 26 56" src="https://user-images.githubusercontent.com/44771837/162167624-e14f5470-77d5-4112-8af5-ae331c6ffac2.png">